### DISCUSSION TITLE
Windows환경에서 14.1 버젼 실행 오류

### DISCUSSION THREAD

**Discussion by CharlesKwonOhjun** [bab2min/Kiwi#121] [Q&A]
안녕하세요?
Windows용 binary를 다운 받아 다음과 같이 처음 실행 하는데 아래와 같은 error가 나옵니다.
directory는 ./ 와 ../ 혹은 절대 경로를 설정해도 같은 결과가 나오고 있습니다.

kiwi-evaluator.exe --model ./ModelGenerator ./eval_data/* --sbg
reading type 'char16_t' failed: iostream stream error

**Comment by bab2min** [comment]
안녕하세요 @CharlesKwonOhjun 님
https://github.com/bab2min/Kiwi/issues/119 에서도 비슷한 오류가 제보된 바 있는데요,
여기에서 제시된 해결책처럼 모델 파일을 받아서 압축을 푸신뒤 실행해보세요~
